tutorials/hyperspectral/RasterStack-RGB-Images-in-R-Using-HSI.Rmd

```diff
@@ -236,61 +211,61 @@ band2Raster <- function(file, band, noDataValue, xMin, yMin, res, crs){
 Now that the function is created, we can create our list of rasters. The list 
 specifies which bands (or dimensions in our hyperspectral dataset) we want to 
 include in our raster stack. Let's start with a typical RGB (red, green, blue) 
-combination. We will use bands 58, 34, and 19. 
+combination. We will use bands 14, 9, and 4 (bands 58, 34, and 19 in a full 
```

Comment:
I really like that you've listed the full set bands throughout.

Reply:
Thanks! Things are a bit more complicated with the subset, but I thought this would be helpful.